src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java

```diff
@@ -0,0 +1,57 @@
+package org.micks.champmaker.championships;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.micks.champmaker.EntityNotFoundException;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class ChampionshipServiceTest {
+
+    @InjectMocks
+    private ChampionshipService championshipService;
+
+    @Mock
+    private ChampionshipRepository championshipRepository;
+
+    @Test
+    void shouldReturnChampionship() throws EntityNotFoundException {
+        // given
+
+        Optional<ChampionshipEntity> mockedChampionship = Optional.of(new ChampionshipEntity("Pomeranian", "Szczecin", "15 August"));
```

Comment:
```suggestion
        Optional<ChampionshipEntity> expectedChampionship = Optional.of(new ChampionshipEntity("Pomeranian", "Szczecin", "15 August"));
```
This object is not mocked, it's real test object and you later do assertions against it's values.

Reply:
Fixed